File: Curs Backend/WebApi/Models/InventoryItem.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Inventory.Models
{
    public class InventoryItem
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string User {  get; set; }

        public string Location { get; set; }

        public int InventoryNumber { get; set; }

        public DateTime CreatedAt { get; set; }

        public DateTime ModifiedAt { get; set; }

        public bool Deleted { get; set; }
    }
}

File: Curs Backend/Inventory/src/app/app-logic/inventory-item.ts
export class InventoryItem {
  id!: number;
  name!: string;
  description!: string;
  user!: string;
  location!: string;
  inventoryNumber!: number;
  createdAt!: Date;
  modifiedAt!: Date;
  deleted!: boolean;

  constructor(item?: Partial<InventoryItem>) {
    console.log(item)
    Object.assign(this, item);
  }
}
